architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -106,6 +106,10 @@ their application build.
 Ecosystem:: S2I encourages a shared ecosystem of images where you can leverage
 best practices for your applications.
 
+Reproducibility:: Produced images can include all inputs used to produce the image, 
+ensuring the image can be reproduced precisely including the versions of build tools 
+and dependencies.
+
```

Comment:
Suggest: "Produced images can include all inputs, including specific versions of build tools and dependencies.  This ensures that the image can be reproduced precisely."